src/utils/clack-utils.ts

```diff
@@ -886,11 +885,14 @@ async function askForWizardLogin(options: {
 
   Sentry.setTag('already-has-sentry-account', hasSentryAccount);
 
-  let wizardHash: string;
+  const nodeFetch = await import('node-fetch').then((m) => m.default);
```

Comment:
we have to dynamically import node-fetch here because we still emit CJS which is not supported in node-fetch@3 anymore. Using node-fetch@2 however is also annoying because there are no types and I wanna avoid relying on old package versions.